cars/car.js

```diff
@@ -1,9 +1,18 @@
 module.exports = Car
 
-function Car(mpg) {
-
+function Car({mpg, gallons, odometer, trips}) {
+this.mpg = 10,
```

Comment:
Don't change the method signature `Car(mpg)` - and make sure to set the `this.mpg = mpg`
The other `this...` vars are being initialized to defaults - not from the functions arguments.